README.md

```diff
@@ -13,44 +13,115 @@ View the [list of supported APIs and Services](https://cloud.google.com/php/docs
 
 If you need support for other Google APIs, please check out the [Google APIs Client Library for PHP](https://github.com/google/google-api-php-client).
 
-## Quick Start
+## Quickstart
```

Comment:
I think the first thing a user should see is an explanation for how to install the packages. This could go even before the `Quickstart` header, so something like we had:

````
We recommend installing individual component packages. A list of available packages can be found on [Packagist](https://packagist.org/search/?q=google%2Fcloud-).

For example:
```sh
$ composer require google/cloud-storage
$ composer require google/cloud-bigquery
$ composer require google/cloud-datastore
```
````

And then the quickstart can follow after that.